src/Elastic.Apm/Model/ExecutionSegmentCommon.cs

```diff
@@ -274,5 +275,46 @@ IReadOnlyCollection<string> applicationNamespaces
 				_ => null
 			};
 		}
+
+		/// <summary>
+		/// Captures an error based on a log
+		/// </summary>
+		/// <param name="errorLog"></param>
+		/// <param name="payloadSender"></param>
+		/// <param name="logger"></param>
+		/// <param name="executionSegment"></param>
+		/// <param name="configSnapshot"></param>
+		/// <param name="enclosingTransaction"></param>
+		/// <param name="parentId"></param>
+		/// <param name="serverInfo"></param>
+		/// <param name="exception"></param>
+		/// <param name="labels"></param>
+		internal static void CaptureLogAsError(ErrorLog errorLog, IPayloadSender payloadSender, IApmLogger logger,
```

Comment:
`CaptureLogAsError` -> `CaptureErrorLog` ?

Reply:
Changed it